src/index.jsx

```diff
@@ -57,6 +57,7 @@ initialize({
         EXPERIMENT_2_VARIANT_2_ID: process.env.EXPERIMENT_2_VARIANT_2_ID || null,
         PREQUERY_SEARCH_EXPERIMENT_ID: process.env.PREQUERY_SEARCH_EXPERIMENT_ID || null,
         PREQUERY_SEARCH_EXPERIMENT_VARIANT_ID: process.env.PREQUERY_SEARCH_EXPERIMENT_VARIANT_ID || null,
+        GETSMARTER_STUDENT_DATA_SHARING_URL: process.env.GETSMARTER_STUDENT_DATA_SHARING_URL || null,
```

Comment:
Please use the GetSmarter URL that is already present in edx-internal
https://github.com/edx/edx-internal/blob/1b40f77bb53c4ea2101af3067842b690ddea46cc/frontends/frontend-app-learner-portal-enterprise/prod_config.yml#L32C3-L32C32